terra-jupyter-hail/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.1.2 - 2023-08-01T15:57:27.329878Z
+
+- Update `terra-jupyter-base` to `1.1.2`
```

Comment:
should we include all the dependency bumps here?  Specifically terra-jupyter-python in this case.

Reply:
I [asked](https://broadinstitute.slack.com/archives/C046XJD9MPW/p1690485376690809) about updating the base image only but never got a response to that specific suggestion.